src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1256,6 +1256,13 @@ object FunctionRegistry {
     "min" -> "Minimum value of interval.",
     "max" -> "Maximum value of interval, non-inclusive.")
 
+  register("dbeta", { (x: Double, a: Double, b: Double) => dbeta(x,a,b)},
+  """
+  Returns a random draw from a beta distribution on [0,1] defined by the parameters a and b.
```

Comment:
Returns the probability density at `x` of a `beta distribution <https://en.wikipedia.org/wiki/Beta_distribution>`__ with parameters a (alpha) and b (beta).